openapi.yaml

```diff
@@ -27062,12 +27197,17 @@ tags:
   - name: Authentication
   - name: Impersonation Settings
   - name: Serverless Cloud Gateways
+  - name: Audit Logs
   - description: DP Nodes
     name: DP Nodes
   - description: DP Certificates
     name: DP Certificates
   - description: Custom Plugin Schemas
     name: Custom Plugin Schemas
+  - description: Config Stores
+    name: Config Stores
+  - description: Config Store Secrets
+    name: Config Store Secrets
```

Comment:
```suggestion
```

Removing as these tags are not used in the provider. I'll fix the upstream tooling